nf_core/pipeline-template/{{cookiecutter.name_noslash}}/conf/uzh.config

```diff
@@ -0,0 +1,19 @@
+/*
+ * --------------------------------------------------------------------------------
+ *  Nextflow config file for use with Singularity on University of Zurich Cluster
+ * --------------------------------------------------------------------------------
+ */
+
+singularity {
+    enabled = true
+}
+
+process {
+    executor = 'slurm'
+}
+
+params {
+  max_memory = 1800.GB
```

Comment:
Impressive!

White space inconsistency alert though 🤓

Reply:
Resolved it :-)